test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/StructuralTypeConfigurationTest.cs

```diff
@@ -188,5 +188,39 @@ public void NamespaceAssignment_AutoAssignsNamespaceToStructuralType_NamespaceAs
             Assert.Equal(expectedNamespace, modelBuilder.EntityType<OrderHeader>().Namespace);
             Assert.Equal(expectedNamespace, modelBuilder.ComplexType<ZipCode>().Namespace);
         }
+
+        /// <summary>
+        /// Tests the Namespace property setter logic with a null or white space value throws an argument exception.
+        /// </summary>
+        [Fact]
+        public void NamespaceAssignment_WithNullOrWhiteSpace_ThrowsArgumentException()
+        {
+            // Arrange
+            ODataConventionModelBuilder modelBuilder = new ODataConventionModelBuilder();
+
+            EntityTypeConfiguration<MyOrder> entityTypeConfiguration = modelBuilder.EntitySet<MyOrder>("orders").EntityType;
+
+            // Act and Assert
+            ExceptionAssert.ThrowsArgument(() => entityTypeConfiguration.Namespace = null, "value");
+            ExceptionAssert.ThrowsArgument(() => entityTypeConfiguration.Namespace = string.Empty, "value");
+            ExceptionAssert.ThrowsArgument(() => entityTypeConfiguration.Namespace = " ", "value");
+        }
+
+        /// <summary>
+        /// Tests the Name property setter logic with a null or white space value throws an argument exception.
+        /// </summary>
+        [Fact]
+        public void NameAssignment_WithNullOrWhiteSpace_ThrowsArgumentException()
```

Comment:
@icnocop Instead of having 3 asserts for 3 different scenarios, we could use:
```csharp
[Theory]
[InlineData(null)]
[InlineData("")
[InlineData(" ")]
public void NameAssignment_WithNullOrWhiteSpace_ThrowsArgumentException(string name)
{
    // ...
    ExceptionAssert.ThrowsArgument(() => entityTypeConfiguration.Name = name, "value");
}
```

Reply:
Fixed. Thank you.